kubernetes/utils/create_from_yaml.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from os import path
+import re
+import sys
+
+from six import iteritems
+
+import yaml
+
+from kubernetes import client
+
+def create_from_yaml(k8s_client, yaml_file, verbose=False, **kwargs):
+    """
+    Perform an action from a yaml file. Pass 1 for verbose to
+    print confirmation information.
+    
+    Input:
+    yaml_file: string. Contains the path to yaml file.
+    k8s_cline: an ApiClient object, initialized with the client args.
+
+    Available parameters for performing the subsequent action:
+    :param async_req bool
+    :param bool include_uninitialized: If true, partially initialized resources are included in the response.
+    :param str pretty: If 'true', then the output is pretty printed.
+    :param str dry_run: When present, indicates that modifications should not be persisted. An invalid or unrecognized dryRun directive will result in an error response and no further processing of the request. Valid values are: - All: all dry run stages will be processed
+    """
+
+    with open(path.abspath(yaml_file)) as f:
+        yml_object = yaml.load(f)
+        #TODO: case of yaml file containing multiple objects
+        api_type, _, api_version = yml_object["apiVersion"].partition("/")
```

Comment:
Please use `group`, `version` and `kind` instead of `api_type`, `api_version` and `action_type`, following the Kubernetes API convention: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md